nixos/modules/services/misc/duckling.nix

```diff
@@ -0,0 +1,64 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.duckling;
+
+  port = cfg.port;
+in {
+  options = {
+    services.duckling = {
+      enable = mkEnableOption "duckling";
+
+      dataDir = mkOption {
```

Comment:
Does this actually need to be configurable? Does it only store logs? Ideally we store logs in `journald` instead, and failing that `/var/log`.

Reply:
I've seen this in a handful of other service configurations, though /var/log may be better than /var/lib

Reply:
Removed due to DynamicUser changes